libraries/core_libs/storage/src/storage.cpp

```diff
@@ -375,6 +376,63 @@ void DbStorage::saveDagBlock(DagBlock const& blk, Batch* write_batch_p) {
   }
 }
 
+// DAG Efficiency
+void DbStorage::savePbftBlockDagEfficiency(uint64_t period, uint16_t efficiency, DbStorage::Batch& batch) {
+  insert(batch, Columns::pbft_block_dag_efficiency, toSlice(period), toSlice(efficiency));
+}
+
+std::vector<uint16_t> DbStorage::getLastIntervalEfficiencies(uint16_t computation_interval) {
+  std::vector<uint16_t> efficiencies;
+
+  auto it =
+      std::unique_ptr<rocksdb::Iterator>(db_->NewIterator(read_options_, handle(Columns::pbft_block_dag_efficiency)));
+  it->SeekToLast();
+  const auto last_period = FromSlice<uint64_t>(it->key().data());
```

Comment:
from my experience ordering is not guaranteed by rocksDB, so this it not good.

Reply:
https://github.com/facebook/rocksdb/wiki/RocksDB-Overview#3-high-level-architecture
So if I am understanding it correctly that means that we have values in column sorted by key